homeassistant/components/alarm_control_panel/strings.json

```diff
@@ -62,5 +62,18 @@
         }
       }
     }
+  },
+  "issues": {
+    "platform_integration_no_support": {
+      "title": "Platform support not supported",
```

Comment:
It might be nice to make these common strings so translators don’t have a lot of duplicate work

Reply:
Does that work with placeholders as well or is it limited to the title in this case?

Reply:
It works